linters/stylelint-header/index.js

```diff
@@ -71,9 +71,11 @@ const rule = createPlugin(
             // Walk comments on root to find if header exists
             let found = false;
             root.walkComments((comment, _idx) => {
+                // Remove any asterisks and whitespace from the texts before comparing
+                const clean = (text) => text.replace(/(\*|\n|\s)/g, '').trim();
                 // If the two strings are at least 98% alike, it's a match
                 if (
-                    compareTwoStrings(comment.text, header) >=
+                    compareTwoStrings(clean(comment.text), clean(header)) >=
```

Comment:
@Westbrook This fix to your stylelint-header tool allows the comparator to match the two strings. Just needed a little pre-cleansing to both strings so it could focus on diffs in the content rather than the formatting.  It didn't have anything to do with the ! important added by Spectrum CSS.